docs/commands.rst

```diff
@@ -0,0 +1,85 @@
+.. _commands:
+
+Commands
+========
+
+``django-pgtrigger`` comes with the ``manage.py pgtrigger`` command. It
+can be called with the following subcommands.
+
+ls
+--
+
+Use ``manage.py pgtrigger ls`` to list all triggers that are actively
+managed by ``django-pgtrigger``. The trigger URI, installation status,
+and whether the trigger is enabled or disabled will be shown.
+
+The following are valid installation status markers:
+
+- ``INSTALLED``: The trigger is installed and up to date
+- ``OUTDATED``: The trigger is installed, but it is outdated. New changes
+  need to be applied with ``manage.py pgtrigger install``.
+- ``UNINSTALLED``: The trigger is not active and is not yet installed.
```

Comment:
Perhaps it'll become clearer when I read the rest of the PR - but "uninstalled" sounds a bit weird. Would "not installed" not be a more apt choice of words?

I suppose, "uninstalled" sounds like something might have been installed, but then later uninstalled.

Not that this is a blocker in any way.. 🤷‍♂️

Reply:
Yea I was trying to go for single words. Do you have any other suggestions?

Reply:
Ah, figured.. Unless `noninstalled` is a word, not really. I'd be cool with uninstalled for now if having two words is more trouble than it's worth. I suppose, if this is still an uppercase constant-like thing, it could be `NOT_INSTALLED`.